packages/block-library/src/search/style.scss

```diff
@@ -43,7 +44,7 @@
 	}
 
 	&.wp-block-search__button-inside .wp-block-search__inside-wrapper {
-		padding: $grid-unit-05;
+		padding: 4px;
 		border: 1px solid $gray-600;
```

Comment:
I missed this one as well 🙈 would be nice to also replace this one with `#949494`, which is a neatly 3:1 UI contrast gray.

Reply:
On it!